plugins/org.jboss.tools.openshift.cdk.server/plugin.xml

```diff
@@ -70,6 +70,22 @@
             stopTimeout="900000"
             supportsRemoteHosts="false"/>
 
+    <serverType
+            behaviourClass="org.jboss.tools.openshift.internal.cdk.server.core.adapter.CRCServerBehaviour"
```

Comment:
As we will one day or another remove CDK support, would be good to have the CRC support have its own namespace ie org.jboss.tools.openshift.internal.crc